src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java

```diff
@@ -74,26 +94,33 @@ public static ItemStack get(Tree wood, int amount)
      */
     public static boolean isBeingSupported(World worldIn, BlockPos pos)
     {
+        int sRangeHor = ConfigTFC.General.FALLABLE.supportBeamRangeHor;
+        int sRangeVert = ConfigTFC.General.FALLABLE.supportBeamRangeVert;
+        int sRangeHorNeg = sRangeHor * -1;
+        int sRangeVertNeg = sRangeVert * -1;
         if (!worldIn.isAreaLoaded(pos.add(-32, -32, -32), pos.add(32, 32, 32)))
         {
-            return true; //If world isn't loaded...
+            return true; // If world isn't loaded...
         }
-        for (BlockPos.MutableBlockPos searchSupport : BlockPos.getAllInBoxMutable(pos.add(-4, -1, -4), pos.add(4, 1, 4)))
+        for (BlockPos.MutableBlockPos searchSupport : BlockPos.getAllInBoxMutable(
+            pos.add(sRangeHorNeg, sRangeVertNeg, sRangeHorNeg), pos.add(sRangeHor, sRangeVert, sRangeHor)))
         {
             IBlockState st = worldIn.getBlockState(searchSupport);
             if (st.getBlock() instanceof BlockSupport)
             {
                 if (((BlockSupport) st.getBlock()).canSupportBlocks(worldIn, searchSupport))
                 {
-                    return true; //Found support block that can support this position
+                    return true; // Found support block that can support this position
                 }
             }
         }
         return false;
     }
 
     /**
-     * This is an optimized way to check for blocks that aren't supported during a cave in, instead of checking every single block individually and calling BlockSupper#isBeingSupported
+     * This is an optimized way to check for blocks that aren't supported during a
+     * cave in, instead of checking every single block individually and calling
+     * BlockSupper#isBeingSupported
```

Comment:
This comment along with the one above it were also victims of IntelliJ code formatting.